x-pack/solutions/observability/plugins/observability/public/navigation_tree.ts

```diff
@@ -439,7 +439,6 @@ function createNavTree({ streamsAvailable }: { streamsAvailable?: boolean }) {
         children: [
           {
             id: 'stack_management', // This id can't be changed as we use it to open the panel programmatically
-            link: 'management',
```

Comment:
This removes the "four squares" icon from the nav item and creates a single click target, which has an icon of an arrow pointing to the right.